### ISSUE TITLE
5 out of 7 Preview option "Markdown Flavors" don't handle div's properly

### ISSUE THREAD

**Issue by taw00** [wereturtle/ghostwriter#481]
Fedora Linux 29
Ghostwriter 1.7.4
Pandoc 2.1.2 (a year old)
Sundown? I don't know what generates Sundown stuff.

NOTE: This bug has been much updated.

Markdown. All flavors are supposed to support html within the markup. If you wrap markdown in DIVs, for example, and if you leave a space between the DIV tag and the markdown, it is supposed to process the markdown as markdown, but in the context of the DIV. I have included screenshots of the initial failure example.

```
<div>

...any markdown formatted text here will be ignored (it should not be)...

</div>
```
SPANs behave as they should, by the way.

After experimenting, I have finally determined it is rendering engine dependent.
**Effects on rendering of markdown between div tags by changing preview settings:**
- Sundown -- Failed.
- **Pandoc -- Success**
- **Pandoc CommonMark -- Success**
- Pandoc GitHub-Flavored Markdown -- Failed.
- Pandoc PHP Markdown Extra -- Failed
- Pandoc Multimarkdown -- Failed (_and failed MMD-compliant markdown table test as well_)
- Pandoc Strict -- Failed

I have attached a markdown test file.
[test-markdown-rendering.zip](https://github.com/wereturtle/ghostwriter/files/3882853/test-markdown-rendering.zip)

Is this more of a Pandoc issue?

**Comment by taw00** [comment]
I did some further testing with more complex `div` usage, like making a two column display using `float: left; width: 45%;` and such. I could change the behavior all over the place moving that `div` around. I haven't figured out the pattern, but it's almost like the div's are all additive. For example, I styled using that first div to `font-style: italics;` and all the text became italics, regardless of where I stuck the additional `div`.

Sounds like html is not being popped onto and off the stack appropriately. Regardless, I could not determine a pattern. Well, not an easily discernible one.

Is there a way to see the raw HTML?
